mirgecom/steppers.py

```diff
@@ -27,9 +27,47 @@
 THE SOFTWARE.
 """
 
+from logpyle import (LogManager, add_general_quantities,
+        add_simulation_quantities, add_run_info, IntervalTimer,
+        set_dt, LogQuantity)
+
+class Pressure(LogQuantity):
```

Comment:
Should we use a MultiLogQuantity here (for min,avg,max)?

Reply:
No, one at a time for now, as discussed.